pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -446,10 +427,9 @@ def log_train_epoch_end_metrics(
 
         # TODO: deprecate 1.0
         else:
-            out = self.__run_legacy_training_epoch_end(
-                num_optimizers, epoch_output, model, is_result_obj, epoch_callback_metrics
+            epoch_log_metrics, epoch_progress_bar_metrics = self.__run_legacy_training_epoch_end(
+                num_optimizers, epoch_output, model, is_result_obj
             )
```

Comment:
we file a follow up issue to fully remove this path

Reply:
No, but we need to clean this asap. I'll try to split it among several PRs as there is a lot of code to remove